packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -267,6 +231,39 @@ exports.onCreateNode = (
       child: node,
     });
   }
+
+  // If not explicitly handled, always fall back to build the page as a "content" page.
+  // This is useful in case the website requires additional MDX pages located in
+  // other file system directories, and thus with different `sourceInstanceName` names.
+  const contentPageFieldData = {
```

Comment:
The fix is to fall back to building the node as a "content" page, if not explicitly handled (e.g. for release notes)